content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Sourcegraph 3.41 release'
+publishDate: 2022-06-21T10:00-07:00
+description: 'Sourcegraph 3.41 introduces running batch changes server-side, file mounting for the batch change container, greater pattern tracking in Code Insights, and admin settings to restrict extensions to those authored by Sourcegraph.'
+tags: [blog, release]
+slug: 'release/3.41'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+changelogItems: 
+  - description: 'Added a new templating variable, batch_change_link, to give more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description. This message would occasionally break CI workflows that parsed the bottom of a message; it can now be moved accordingly.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/35319
+    category: Batch Changes
+  - description: Added `sort` and `limit` filters for capture group insights. This gives Code Insights users more control over which series are displayed.
+    url: https://github.com/sourcegraph/sourcegraph/pull/34611
+    category: Code Insights
+  - description: 'Commit and diff search: The hard limit of 50 repositories has been removed, so you can now run broader searches. Long-running searches will continue running until the timeout is hit.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/36486
+    category: Search
+  - description: 'Zoekt-indexserver has a new debug landing page, /debug, which exposes information about the queue, the list of indexed repositories, and the list of assigned repositories for easier visual debugging. Admins can reach the debug landing page by going to the site admin view and selecting Instrumentation > indexed-search-indexer > Debug.'
+    url: https://github.com/sourcegraph/zoekt/pull/346
+    category: Admin
+---
+ 
+Sourcegraph 3.41 is now available! Here are some highlights from this release:
+
+## Running batch changes server-side is moving to beta
+
+Until now, you had to run `src-cli` locally to create your batch changes. `src-cli` pulls repo archives through Sourcegraph, and runs your batch spec to create diffs. For large amounts of repositories or resource-intensive code rewrites, running `src-cli` locally could take an impractical amount of time or be brittle.
+
+Today, we're announcing that you can now run [batch changes server-side (beta)](https://docs.sourcegraph.com/batch_changes/explanations/server_side). 
+
+Setting up running batch changes server-side allows you to:
```

Comment:
@iskyOS - Should this just be "Using/setting up batch changes server-side allows you to:"

Reply:
Yes - definitely.

```suggestion
Running batch changes server-side allows you to:
```